### PR TITLE
Add instructions on creating Canvas rubrics on the publish Project page

### PR DESCRIPTION
**Pull request by sadleb**
### Description

This is a follow-up to https://app.asana.com/0/1174274412967132/1191666190028013 that adds some basic instructions to remind Designers of what to do in order to create and attach a Canvas rubric to a project.

### Motivation and Context

Note: Canvas rubrics used for TA grading are separate from rubric information inserted into the Project content itself for Fellows to reference. See here for the latter: https://app.asana.com/0/1174274412967132/1195096839523739

### Screenshots (if appropriate):
image

### Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

### Checklist:

- [ ] My code follows the code style of this project.
- [ ] I have added neccessary labels
- [ ] My PR title follows the convention in the guide
- [ ] I have assigned myself to the PR
- [ ] I have requested reviews from team members
- [ ] I have created a task for reviewers on PM tool
- [ ] All new and existing tests passed.

### CODE REVIEW

File: app/views/base_course_custom_content_versions/new.html.erb [hunk 1]
```diff
@@ -28,4 +27,28 @@
 
     </div>
   </div>
+
+  <br/><br/>
+
+  <div class="row">
+    <div class="col-sm-6">
+      <h5>Rubric instructions</h5>
+      <p>To add a rubric to a Project in Canvas in order for Teaching Assignments (TAs) to be able to use it to grade, do the following:</p>
```
Comment (rshipp) [inline app/views/base_course_custom_content_versions/new.html.erb:36]: s/Assignments/Assistants

**Comment by jessouyangbraven** [comment]
Does it make sense to move the instructions to a doc/wiki that doesn't require a code review/merge to update and link to that instead?

I do think people are going to read the instructions more if they're on the actual page, so I'm fine with this too if we don't expect this to change often.

File: app/views/base_course_custom_content_versions/new.html.erb (hunk 1 above)
Comment (sadleb) [reply app/views/base_course_custom_content_versions/new.html.erb:36]: Lol, good catch. I told ya’ll my brain was dead today!

**Comment by sadleb** [comment]
> Does it make sense to move the instructions to a doc/wiki that doesn't require a code review/merge to update and link to that instead?
> 
> I do think people are going to read the instructions more if they're on the actual page, so I'm fine with this too if we don't expect this to change often.

I think it's better to have it here for now b/c a) like you said, it's more likely they'll read it here b) if we make changes to how this works, I think it's more likely we'll remember to update this than a wiki/doc c) we don't have training docs yet for Highlander features / flows and I don't think it's worth trying to figure out what that looks like so we can link to it at the moment. We can remove and/or link to docs once we're further along in operationalizing overall Highlander execution.

**Pull request merged**